ios/NativeModules/Multipeer.m

```diff
@@ -0,0 +1,88 @@
+#import "Multipeer.h"
+
+@interface Multipeer()
+
+@end
+
+@implementation Multipeer
+
+RCT_EXPORT_MODULE();
+
+RCT_EXPORT_METHOD(init:(NSString *)name address:(NSString *)address avatar:(NSString *)avatar) {
```

Comment:
Okay, you see I init with some Textile specific variables. The idea here was that when detected by another device, that device could show the real Textile contact info.